
from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_telaGraficodePresenca(object):
    def setupUi(self, telaGraficodePresenca):
        telaGraficodePresenca.setObjectName("telaGraficodePresenca")
        telaGraficodePresenca.resize(650, 320)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Fixed, QtWidgets.QSizePolicy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(telaGraficodePresenca.sizePolicy().hasHeightForWidth())
        telaGraficodePresenca.setSizePolicy(sizePolicy)
        telaGraficodePresenca.setMinimumSize(QtCore.QSize(650, 320))
        telaGraficodePresenca.setMaximumSize(QtCore.QSize(650, 320))
        self.centralwidget = QtWidgets.QWidget(telaGraficodePresenca)
        self.centralwidget.setObjectName("centralwidget")
        self.gridLayout = QtWidgets.QGridLayout(self.centralwidget)
        self.gridLayout.setContentsMargins(5, 5, 5, 0)
        self.gridLayout.setVerticalSpacing(0)
        self.gridLayout.setObjectName("gridLayout")
        self.frame_7 = QtWidgets.QFrame(self.centralwidget)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(32)
        sizePolicy.setHeightForWidth(self.frame_7.sizePolicy().hasHeightForWidth())
        self.frame_7.setSizePolicy(sizePolicy)
        self.frame_7.setMaximumSize(QtCore.QSize(16777215, 32))
        self.frame_7.setFrameShape(QtWidgets.QFrame.NoFrame)
        self.frame_7.setFrameShadow(QtWidgets.QFrame.Raised)
        self.frame_7.setObjectName("frame_7")
        self.gridLayout_6 = QtWidgets.QGridLayout(self.frame_7)
        self.gridLayout_6.setContentsMargins(0, 0, 0, 4)
        self.gridLayout_6.setHorizontalSpacing(6)
        self.gridLayout_6.setVerticalSpacing(0)
        self.gridLayout_6.setObjectName("gridLayout_6")
        self.BTN_FECHAR = QtWidgets.QToolButton(self.frame_7)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Fixed, QtWidgets.QSizePolicy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.BTN_FECHAR.sizePolicy().hasHeightForWidth())
        self.BTN_FECHAR.setSizePolicy(sizePolicy)
        self.BTN_FECHAR.setMinimumSize(QtCore.QSize(29, 29))
        self.BTN_FECHAR.setMaximumSize(QtCore.QSize(29, 29))
        icon = QtGui.QIcon()
        icon.addPixmap(QtGui.QPixmap("icons/Exit.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.BTN_FECHAR.setIcon(icon)
        self.BTN_FECHAR.setIconSize(QtCore.QSize(25, 25))
        self.BTN_FECHAR.setObjectName("BTN_FECHAR")
        self.gridLayout_6.addWidget(self.BTN_FECHAR, 0, 1, 1, 1)
        spacerItem = QtWidgets.QSpacerItem(40, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.gridLayout_6.addItem(spacerItem, 0, 0, 1, 1)
        self.gridLayout.addWidget(self.frame_7, 0, 0, 1, 1)
        self.frame = QtWidgets.QFrame(self.centralwidget)
        self.frame.setFrameShape(QtWidgets.QFrame.NoFrame)
        self.frame.setFrameShadow(QtWidgets.QFrame.Raised)
        self.frame.setObjectName("frame")
        self.gridLayout_2 = QtWidgets.QGridLayout(self.frame)
        self.gridLayout_2.setContentsMargins(0, 0, 0, 0)
        self.gridLayout_2.setSpacing(0)
        self.gridLayout_2.setObjectName("gridLayout_2")
        self.frame_2 = QtWidgets.QFrame(self.frame)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Expanding)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.frame_2.sizePolicy().hasHeightForWidth())
        self.frame_2.setSizePolicy(sizePolicy)
        self.frame_2.setFrameShape(QtWidgets.QFrame.NoFrame)
        self.frame_2.setFrameShadow(QtWidgets.QFrame.Raised)
        self.frame_2.setObjectName("frame_2")
        self.gridLayout_4 = QtWidgets.QGridLayout(self.frame_2)
        self.gridLayout_4.setContentsMargins(2, 0, 0, 0)
        self.gridLayout_4.setSpacing(0)
        self.gridLayout_4.setObjectName("gridLayout_4")
        self.GW_MOSTRAR_GRAFICO_ = QtWidgets.QGraphicsView(self.frame_2)
        self.GW_MOSTRAR_GRAFICO_.setObjectName("GW_MOSTRAR_GRAFICO_")
        self.gridLayout_4.addWidget(self.GW_MOSTRAR_GRAFICO_, 0, 0, 1, 1)
        self.gridLayout_2.addWidget(self.frame_2, 0, 0, 1, 1)
        self.frame_3 = QtWidgets.QFrame(self.frame)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Expanding)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.frame_3.sizePolicy().hasHeightForWidth())
        self.frame_3.setSizePolicy(sizePolicy)
        self.frame_3.setFrameShape(QtWidgets.QFrame.NoFrame)
        self.frame_3.setFrameShadow(QtWidgets.QFrame.Raised)
        self.frame_3.setObjectName("frame_3")
        self.gridLayout_5 = QtWidgets.QGridLayout(self.frame_3)
        self.gridLayout_5.setContentsMargins(6, 0, 2, 0)
        self.gridLayout_5.setSpacing(0)
        self.gridLayout_5.setObjectName("gridLayout_5")
        self.GW_COMPARAR_GRAFICO = QtWidgets.QGraphicsView(self.frame_3)
        self.GW_COMPARAR_GRAFICO.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.GW_COMPARAR_GRAFICO.setObjectName("GW_COMPARAR_GRAFICO")
        self.gridLayout_5.addWidget(self.GW_COMPARAR_GRAFICO, 0, 0, 1, 1)
        self.gridLayout_2.addWidget(self.frame_3, 0, 1, 1, 1)
        self.frame_4 = QtWidgets.QFrame(self.frame)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Expanding)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.frame_4.sizePolicy().hasHeightForWidth())
        self.frame_4.setSizePolicy(sizePolicy)
        self.frame_4.setMinimumSize(QtCore.QSize(0, 50))
        self.frame_4.setMaximumSize(QtCore.QSize(16777215, 50))
        self.frame_4.setFrameShape(QtWidgets.QFrame.NoFrame)
        self.frame_4.setFrameShadow(QtWidgets.QFrame.Raised)
        self.frame_4.setObjectName("frame_4")
        self.gridLayout_3 = QtWidgets.QGridLayout(self.frame_4)
        self.gridLayout_3.setContentsMargins(0, 0, 0, 0)
        self.gridLayout_3.setSpacing(0)
        self.gridLayout_3.setObjectName("gridLayout_3")
        self.frame_5 = QtWidgets.QFrame(self.frame_4)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Expanding)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.frame_5.sizePolicy().hasHeightForWidth())
        self.frame_5.setSizePolicy(sizePolicy)
        self.frame_5.setFrameShape(QtWidgets.QFrame.NoFrame)
        self.frame_5.setFrameShadow(QtWidgets.QFrame.Raised)
        self.frame_5.setObjectName("frame_5")
        self.gridLayout_7 = QtWidgets.QGridLayout(self.frame_5)
        self.gridLayout_7.setContentsMargins(0, 0, 0, 0)
        self.gridLayout_7.setSpacing(0)
        self.gridLayout_7.setObjectName("gridLayout_7")
        self.frame_6 = QtWidgets.QFrame(self.frame_5)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Expanding)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.frame_6.sizePolicy().hasHeightForWidth())
        self.frame_6.setSizePolicy(sizePolicy)
        self.frame_6.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.frame_6.setFrameShadow(QtWidgets.QFrame.Raised)
        self.frame_6.setObjectName("frame_6")
        self.gridLayout_8 = QtWidgets.QGridLayout(self.frame_6)
        self.gridLayout_8.setContentsMargins(0, 0, 0, 0)
        self.gridLayout_8.setSpacing(0)
        self.gridLayout_8.setObjectName("gridLayout_8")
        self.BTN_MOSTRAR_GRAFICO = QtWidgets.QToolButton(self.frame_6)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.BTN_MOSTRAR_GRAFICO.sizePolicy().hasHeightForWidth())
        self.BTN_MOSTRAR_GRAFICO.setSizePolicy(sizePolicy)
        self.BTN_MOSTRAR_GRAFICO.setMinimumSize(QtCore.QSize(0, 30))
        self.BTN_MOSTRAR_GRAFICO.setMaximumSize(QtCore.QSize(134, 16777215))
        icon1 = QtGui.QIcon()
        icon1.addPixmap(QtGui.QPixmap("icons/grafico.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.BTN_MOSTRAR_GRAFICO.setIcon(icon1)
        self.BTN_MOSTRAR_GRAFICO.setToolButtonStyle(QtCore.Qt.ToolButtonTextBesideIcon)
        self.BTN_MOSTRAR_GRAFICO.setObjectName("BTN_MOSTRAR_GRAFICO")
        self.gridLayout_8.addWidget(self.BTN_MOSTRAR_GRAFICO, 0, 1, 1, 1)
        spacerItem1 = QtWidgets.QSpacerItem(89, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.gridLayout_8.addItem(spacerItem1, 0, 0, 1, 1)
        spacerItem2 = QtWidgets.QSpacerItem(40, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.gridLayout_8.addItem(spacerItem2, 0, 2, 1, 1)
        self.gridLayout_7.addWidget(self.frame_6, 0, 0, 1, 1)
        self.frame_8 = QtWidgets.QFrame(self.frame_5)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Expanding)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.frame_8.sizePolicy().hasHeightForWidth())
        self.frame_8.setSizePolicy(sizePolicy)
        self.frame_8.setFrameShape(QtWidgets.QFrame.NoFrame)
        self.frame_8.setFrameShadow(QtWidgets.QFrame.Raised)
        self.frame_8.setObjectName("frame_8")
        self.gridLayout_9 = QtWidgets.QGridLayout(self.frame_8)
        self.gridLayout_9.setContentsMargins(0, 0, 0, 0)
        self.gridLayout_9.setSpacing(0)
        self.gridLayout_9.setObjectName("gridLayout_9")
        self.BTN_COMPARAR_GRAFICO = QtWidgets.QToolButton(self.frame_8)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.BTN_COMPARAR_GRAFICO.sizePolicy().hasHeightForWidth())
        self.BTN_COMPARAR_GRAFICO.setSizePolicy(sizePolicy)
        self.BTN_COMPARAR_GRAFICO.setMinimumSize(QtCore.QSize(0, 30))
        self.BTN_COMPARAR_GRAFICO.setMaximumSize(QtCore.QSize(86, 30))
        self.BTN_COMPARAR_GRAFICO.setIcon(icon1)
        self.BTN_COMPARAR_GRAFICO.setToolButtonStyle(QtCore.Qt.ToolButtonTextBesideIcon)
        self.BTN_COMPARAR_GRAFICO.setObjectName("BTN_COMPARAR_GRAFICO")
        self.gridLayout_9.addWidget(self.BTN_COMPARAR_GRAFICO, 0, 1, 1, 1)
        spacerItem3 = QtWidgets.QSpacerItem(114, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.gridLayout_9.addItem(spacerItem3, 0, 2, 1, 1)
        spacerItem4 = QtWidgets.QSpacerItem(40, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.gridLayout_9.addItem(spacerItem4, 0, 0, 1, 1)
        self.gridLayout_7.addWidget(self.frame_8, 0, 1, 1, 1)
        self.gridLayout_3.addWidget(self.frame_5, 0, 0, 1, 1)
        self.gridLayout_2.addWidget(self.frame_4, 1, 0, 1, 2)
        self.gridLayout.addWidget(self.frame, 1, 0, 1, 1)
        telaGraficodePresenca.setCentralWidget(self.centralwidget)

        self.retranslateUi(telaGraficodePresenca)
        self.BTN_FECHAR.clicked.connect(telaGraficodePresenca.close) # type: ignore
        QtCore.QMetaObject.connectSlotsByName(telaGraficodePresenca)

    def retranslateUi(self, telaGraficodePresenca):
        _translate = QtCore.QCoreApplication.translate
        telaGraficodePresenca.setWindowTitle(_translate("telaGraficodePresenca", "GRÁFICOS DE PRESENÇA"))
        self.BTN_FECHAR.setText(_translate("telaGraficodePresenca", "..."))
        self.BTN_MOSTRAR_GRAFICO.setText(_translate("telaGraficodePresenca", " MOSTRAR GRÁFICO"))
        self.BTN_COMPARAR_GRAFICO.setText(_translate("telaGraficodePresenca", "COMPARAR"))


if __name__ == "__main__":
    import sys
    app = QtWidgets.QApplication(sys.argv)
    telaGraficodePresenca = QtWidgets.QMainWindow()
    ui = Ui_telaGraficodePresenca()
    ui.setupUi(telaGraficodePresenca)
    telaGraficodePresenca.show()
    sys.exit(app.exec_())
